lxd/instance/drivers/driver_lxc.go

```diff
@@ -3397,6 +3397,10 @@ func (d *lxc) Delete(force bool) error {
 				if err != nil {
 					return err
 				}
+				err = c.UpdateBackupFile()
```

Comment:
@feileacan please can you move this down 1 line so there is a gap from the previous if statement. Thanks